front/lib/api/assistant/messages.ts

```diff
@@ -579,7 +579,22 @@ export async function fetchMessageInConversation(
         model: AgentMessage,
         as: "agentMessage",
         required: false,
+        include: [
```

Comment:
How bad does it impact perfs? I'm always scared by doin joins with sequelize 🙈

Reply:
I think we should fetch separately